### ISSUE TITLE
WAVE tool reports 'Empty heading' error on collections page

### ISSUE THREAD

**Issue by h-parekh** [samvera-labs/nurax-pre2023#224]
### Descriptive summary
Hyrax Version: 2.1.0.beta1 
Hyrax Branch: v2.1.0.beta1_qa 
Hyrax Revision: 8fe35f54e067fd499a4467b9ccbefa48233d43a2

If I click a specific collection, there is an h2 element above the search box that complains about 'Empty heading'. 
This is non-compliance with section 508 as well as WCAG standards.

```html
<div class="hyc-blacklight hyc-bl-title">
			<h2>
			</h2>
</div>
```

#### What It Means
A heading contains no content.
#### Why It Matters
Some users, especially keyboard and screen reader users, often navigate by heading elements. An empty heading will present no information and may introduce confusion.
#### How to Fix It
Ensure that all headings contain informative content.

### Expected behavior
WAVE tool should not report this error

### Actual behavior
WAVE tool reports an error

### Steps to reproduce the behavior
1. Install the WAVE [extension for chrome](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
2.  Load homepage
3. click one of the records displayed in the explore collections tab. You should be on URL: https://<hostname>/collections/<collection_id>
4. Click on the WAVE tool's icon and you will see the error on the 'Details' tab of WAVE tool

NOTE: You can reproduce this on Nurax as well as vanilla hyrax, but there will be other accessibility errors in Nuraax that have been fixed in Vanilla Hyrax.
